src/frontend/js/utils/test/factories/joanie.ts

```diff
@@ -58,6 +64,17 @@ export const EnrollmentFactory = factory((): Enrollment => {
   };
 });
 
+export const EnrollmentLightFactory = factory((): EnrollmentLight => {
+  return {
+    id: faker.string.uuid(),
+    course_run: CourseRunWithCourseFactory().one(),
+    is_active: true,
```

Comment:
What do you think to use a random boolean value ?

Reply:
I prefer always have the same default value.
Most of the time we'll want active enrollment, it'll be easier not to add `{is_active: true}` everywhere.

Reply:
Indeed !